satpy/tests/test_composites.py

```diff
@@ -250,27 +250,34 @@ def test_more_than_three_datasets(self):
         with pytest.raises(ValueError, match="Expected 3 datasets, got 4"):
             comp((self.ds1, self.ds2, self.ds3, self.ds1), optional_datasets=(self.ds4_big,))
 
-    def test_self_sharpened_no_high_res(self):
+    @pytest.mark.parametrize("dtype", [np.float32, np.float64])
```

Comment:
I don't think type play any role here?

Reply:
The data passed in this case is `float64` so I'd say it's better to test also with `float32` to be sure.

Reply:
Nevermind. Removed in https://github.com/pytroll/satpy/pull/2954/commits/3f1076ae94af61c375be206d260b3b96d86520b9